tests/GitElephant/TestCase.php

```diff
@@ -125,17 +126,13 @@ protected function tearDown(): void
      *
      * @return void
      */
-    protected function addFile($name, $folder = null, $content = null, $repository = null)
+    protected function addFile($name, $folder = null, $content = null, $repository = null): void
     {
-        if (is_null($repository)) {
-            $path = $this->path;
-        } else {
-            $path = $repository->getPath();
-        }
+        $path = is_null($repository) ? $this->path : $repository->getPath();
         $filename = $folder == null ?
             $path . DIRECTORY_SEPARATOR . $name : $path . DIRECTORY_SEPARATOR . $folder . DIRECTORY_SEPARATOR . $name;
```

Comment:
Suggestion for readability improvement:
```suggestion
        $filename = $folder == null 
            ? $path . DIRECTORY_SEPARATOR . $name 
            : $path . DIRECTORY_SEPARATOR . $folder . DIRECTORY_SEPARATOR . $name;
```